source/statements.tex

```diff
@@ -1001,8 +1001,9 @@
 declaration.
 If control enters the declaration concurrently while the variable is
 being initialized, the concurrent execution shall wait for completion
-of the initialization.\footnote{The implementation must not introduce
-any deadlock around execution of the initializer. Deadlocks might
+of the initialization. The implementation shall not introduce
+any deadlock around execution of the initializer.
+\footnote{Deadlocks might
 still be caused by the program logic; the implementation need only
 avoid deadlocks due to its own synchronization operations.} If control
```

Comment:
I'm nervous about this; the phrasing of this is vague, and "shall"ing it might add a requirement that's stronger than what we meant. I think rather than a "shall" here, we want to point out that introducing a deadlock would not satisfy "as if". How about this:

```suggestion
of the initialization.
\begin{note}
A conforming implementation cannot introduce
any deadlock around execution of the initializer.
Deadlocks might still be caused by the program logic;
the implementation need only avoid deadlocks
due to its own synchronization operations.
\end{note}
If control
```

Reply:
done